source/time_tables/departure_boards.cpp

```diff
@@ -65,6 +66,20 @@ static bool is_terminus_for_all_stop_times(const std::vector<routing::datetime_s
     return !stop_times.empty();
 }
 
+bool update_display_information(bool& vj_found,
+                                const navitia::type::StopTime* st,
+                                pbnavitia::PtDisplayInfo* pt_display_information,
+                                PbCreator& pb_creator) {
```

Comment:
i believe `vj_found` is no longer needed
```suggestion
bool update_display_information(const navitia::type::StopTime* st,
                                pbnavitia::PtDisplayInfo* pt_display_information,
                                PbCreator& pb_creator) {
```